frontend/components/Covid19.js

```diff
@@ -0,0 +1,5 @@
+import Covid19 from '../pages/covid19'
+import React, { Component } from 'react'
+import Link from 'next/link'
+
+export default RoyalPerthHospital
```

Comment:
This doesn't look right why are we importing a bunch or stuff and not doing anything with it and exporting something that doesn't exist?